### PR TITLE
rebar3 3.6.0

### PR DESCRIPTION
**Pull request by getong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,47 +52,48 @@ Read from https://github.com/erlang/otp/releases for each tag description as rel
...
+compiler-7.2      et-1.6.2              os_mon-2.4.5        stdlib-3.5
+crypto-4.3        eunit-2.3.6           otp_mibs-1.2        syntax_tools-2.1.5
+debugger-4.2.5    ftp-1.0               parsetools-2.1.7    tftp-1.0
+dialyzer-3.3      hipe-3.18             public_key-1.6      tools-3.0
+diameter-2.1.5    inets-7.0             reltool-0.7.6       wx-1.8.4
+edoc-0.9.3        kernel-6.0            runtime_tools-1.13  xmerl-1.3.17
+eldap-1.2.4       megaco-3.18.3         sasl-3.2
+erl_docgen-0.8    mnesia-4.15.4         snmp-5.2.11
+root@ed434f6c1081:/# ls /usr/local/lib/erlang/lib/ | wc -l
+     38      38     476 -
```
Comment (c0b) [inline README.md]: did you actually run this line?  the `wc -l` is supposed to get number of lines only, not including number of words / number of characters;   should be one number only

File: README.md [hunk 2]
```diff
@@ -6,33 +6,33 @@
...
-"BINDIR=/usr/local/lib/erlang/erts-9.0/bin","PROGNAME=erl",
-"EMU=beam","OTP_VERSION=20.0","HOME=/root",
+"BINDIR=/usr/local/lib/erlang/erts-10.0/bin","PROGNAME=erl",
+"EMU=beam","OTP_VERSION=21.0","HOME=/root",
 "HOSTNAME=9b1e7f4d7206"]
-3> 'hello_юникод_世界'.                                   % Erlang20 now support unicode in atom
+3> 'hello_юникод_世界'.                                   % Erlang21 now support unicode in atom
 'hello_юникод_世界'
 4> io:format("~tp~n", [{'hello_юникод', <<"Hello, 世界; юникод"/utf8>>, "Hello, 世界; юникод"}]).
 {'hello_юникод',<<"Hello, 世界; юникод"/utf8>>,"Hello, 世界; юникод"}
```
Comment (c0b) [inline README.md]: usually with new major version, I'd like to use the README to demo case one piece of brand new feature, like the Unicode atom in Erlang 20;  so for Erlang 21, what do you think is a prominent new feature only available in 21 ?

from http://erlang.org/pipermail/erlang-questions/2018-June/095691.html how do you think the `catch C:R:Stacktrace` syntax ? like

```erlang
1> try 1/0 catch C:R:Stacktrace -> io:format("caught: ~tp", [{C,R,Stacktrace}]) end.
caught: {error,badarith,
               [{erlang,'/',[1,0],[]},
                {erl_eval,do_apply,6,[{file,"erl_eval.erl"},{line,681}]},
                {erl_eval,try_clauses,8,[{file,"erl_eval.erl"},{line,911}]},
                {shell,exprs,7,[{file,"shell.erl"},{line,686}]},
                {shell,eval_exprs,7,[{file,"shell.erl"},{line,642}]},
                {shell,eval_loop,3,[{file,"shell.erl"},{line,627}]}]}ok
```

Comment (garazdawi) [reply README.md]: If I get a vote, I'd say logger is the most prominent new feature.

**Comment by c0b** [comment]
now   Looks Good To Merge

**Pull request merged**